test/functional/pdo_sqlsrv/pdo_utf8_conn.phpt

```diff
@@ -21,7 +21,7 @@ if ($c !== false) {
 ?>
 --EXPECTREGEX--
 
-Fatal error: Uncaught PDOException: SQLSTATE\[(28000|08001|HYT00)\]: .*\[Microsoft\]\[ODBC Driver 1[0-9] for SQL Server\](\[SQL Server\])?(Named Pipes Provider: Could not open a connection to SQL Server \[2\]\. |Login timeout expired|Login failed for user 'sa'\.) in .+(\/|\\)pdo_utf8_conn\.php:[0-9]+
+Fatal error: Uncaught PDOException: SQLSTATE\[(28000|08001|HYT00)\]: .*\[Microsoft\]\[ODBC Driver 1[0-9] for SQL Server\](\[SQL Server\])?(Named Pipes Provider: Could not open a connection to SQL Server \[2\]\. |TCP Provider: Error code 0x2AF9|Login timeout expired|Login failed for user 'sa'\.) in .+(\/|\\)pdo_utf8_conn\.php:[0-9]+
 Stack trace:
```

Comment:
so basically you added yet another possible error message?

Reply:
Yes, that's the one that shows up on mac. It corresponds to WSAHOST_NOT_FOUND (no such host is known).